sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -122,15 +115,13 @@ public bool IsClosed
                 Argument.AssertNotNullOrWhiteSpace(entityPath, nameof(entityPath));
                 connection.ThrowIfClosed();
 
-                options = options?.Clone() ?? new ServiceBusSenderOptions();
+                options = options ?? new ServiceBusSenderOptions();
```

Comment:
Even though it is removed from the public surface, why would we stop cloning it?

Reply:
It has no properties now.. I was going to just leave it, but I was getting an error about it needing to be static. I will just override this error and add back the Clone.